### PR TITLE
Allow both CORINFO_FLG_INTRINSIC and CORINFO_FLG_JIT_INTRINSIC to be set on potential intrinsics

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

**Comment by jkotas** [comment]
https://github.com/dotnet/corert/pull/4756/#discussion_r145458794

File: src/jit/importer.cpp [hunk 1]
```diff
@@ -3289,14 +3289,13 @@ GenTreePtr Compiler::impInitializeArrayIntrinsic(CORINFO_SIG_INFO* sig)
 //    clsHnd - handle for the intrinsic method's class
 //    method - handle for the intrinsic method
 //    sig    - signature of the intrinsic method
+//    methodFlags - CORINFO_XXX flagas of the intrinsic method
```
Comment (AndyAyersMS) [inline src/jit/importer.cpp]: typo: flags

**Comment by jkotas** [comment]
@dotnet-bot test CentOS7.1 x64 Release Priority 1 Build and Test please

**Comment by jkotas** [comment]
CentOS failure is #14581

**Pull request merged**